includes/scripts.php

```diff
@@ -94,11 +94,6 @@ function pmpro_enqueue_scripts() {
  * Enqueue admin JavaScript and CSS
  */
 function pmpro_admin_enqueue_scripts() {
-	// Only enqueue PMPro admin scripts on our own pages.
-	if ( ! isset( $_GET['page'] ) || 0 !== strpos( $_GET['page'], 'pmpro' ) ) {
-		return;
-	}
-
```

Comment:
Let's pull this change out in favor of https://github.com/strangerstudios/paid-memberships-pro/pull/1745